package.json

```diff
@@ -201,5 +202,8 @@
     "webpack-stream": "^5.2.1",
     "webpackbar": "^3.1.5",
     "whatwg-fetch": "^3.0.0"
+  },
+  "dependencies": {
+    "invariant": "^2.2.4"
```

Comment:
Why is this not under `devDependencies`?

Reply:
We should really move to specifying dependencies with more clarity. Anything bundled into the app in production/build _should_ be included in our `dependencies`, not `devDependencies`. It _kinda_ doesn't matter right now as Site Kit isn't included by other modules, but it's best practice and clarifies the intent/usage of the code we're including.

I think in the past everything just went into `devDependencies` because it doesn't _effectively_ make a difference, but it's also an easy change to make and I'd argue we should be clearer about our intent with dependencies going forward.

Reply:
@tofumatt That's unrelated to this change though, and is more than just putting `invariant` here. Let's move this to `devDependencies` like the others in this PR, and open a separate issue about splitting _all_ actual `dependencies` out, as you suggest.

Reply:
Fair enough; I've rolled it back for now 😄